provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/PractitionerRoleWrapper.java

```diff
@@ -0,0 +1,14 @@
+package gov.va.api.health.providerdirectory.service;
+
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder
+public final class PractitionerRoleWrapper {
```

Comment:
Since each `Wrapper` class is only used in the corresponding `Controller` package, let's move them. For example, move this one to `gov.va.api.health.providerdirectory.service.controller.practitionerrole`